redirects.yaml

```diff
@@ -264,7 +264,7 @@ contact/whitepaper/?: "/blog/white-papers/"
 containers/docker-ubuntu/?: "/containers"
 containers/kubernetes/?: "/kubernetes"
 containers/lxd/?: "/lxd"
-core/smartstart(?P<path>.*)/?: "/core/services{path}"
+core/internet-of-things/appstore/guide/?: "/core/services/guide/"
```

Comment:
There is no such thing as /core/internet-of-things. Do you mean /core/smartstart/guide?

Reply:
mm I am not sure, it's what's been requested in the brief

Reply:
Removed. They are not sure we want it, so we'll not do it for now.